pom.xml

```diff
@@ -152,6 +151,23 @@
                     </filesets>
                 </configuration>
             </plugin>
+            <plugin>
+                <groupId>net.revelc.code.formatter</groupId>
+                <artifactId>formatter-maven-plugin</artifactId>
+                <version>2.11.0</version>
+                <configuration>
+                    <configFile>eclipse/VaadinJavaConventions.xml</configFile>
```

Comment:
Isn't the formatter spec required to be in the repo, or how is it located?

Reply:
This is apparently what the flow buildtools dependency provides, see https://github.com/vaadin/flow/blob/master/build-tools/pom.xml#L19

Reply:
Right, `flow-buildtools` includes `../eclipse` in Flow. My bad.